lonestar/analytics/cpu/k-truss/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+app_analy_cpu(K-Truss k-truss)
+add_custom_command(
```

Comment:
Does anyone know why we have bmk-related code in the CMakeLists.txt for k-truss? Is this deliberate?

Reply:
The bmk related code in CMake here was introduced by Sree to run through configurations for collecting performance numbrers. I think we should remove this for the release.

Reply:
This was added by Sree. We should see if we can make it consistent with the other lonestar apps in a separate issue/commit.

Reply:
Noted. I will remove it.

Reply:
Please do not do this in this commit or pull-request. Renaming is a big change - get that out of way. Anything else should be consistent with what was there before the renaming.